controller/api/destination/server.go

```diff
@@ -353,6 +337,66 @@ func (s *server) GetProfile(dest *pb.GetDestination, stream pb.Destination_GetPr
 	return nil
 }
 
+// sendEndpointProfile will send a DestinationProfile response that contains an
+// endpoint to the client. It accepts the gRPC stream object an optional pod and
```

Comment:
missing commas in this comment, I think

Reply:
yup, I can see how "(...) that contains an endpoint to the client" can be confusing without the comma.

Reply:
Yup, quite a few commas. Thanks for bringing this up, I added some commas in but after a bit more thought, I ended up rewriting the whole comment.

I think the comment was a bit too hard to read and included too much information that could be inferred from the function signature. With the new change, I wanted to highlight that the pod argument is optional and how the profile sent back differs based on whether this argument is provided. This should hopefully make everything a bit more clear?